sonar-docs/analysis/scan/sonarscanner-for-msbuild.md

```diff
@@ -18,9 +18,13 @@ url: /analysis/scan/sonarscanner-for-msbuild/
 | [.NET Core Global Tool](https://www.nuget.org/packages/dotnet-sonarscanner)
 <!-- /sonarcloud -->
 
-The SonarScanner for MSBuild is the recommended way to launch an analysis for projects/solutions using MSBuild or dotnet command as a build tool. It is the result of a [collaboration between SonarSource and Microsoft](http://www.sonarqube.org/announcing-sonarqube-integration-with-msbuild-and-team-build/).
+[[info]]
+| Since version 5.0, the SonarScanner for MSBuild is now the SonarScanner for .NET. 
+| documentation is updated with that new name, artifacts and links will remain with the old name for now.
+
+The SonarScanner for .NET is the recommended way to launch an analysis for projects/solutions using .NET or dotnet command as a build tool. It is the result of a [collaboration between SonarSource and Microsoft](http://www.sonarqube.org/announcing-sonarqube-integration-with-msbuild-and-team-build/).
 
-SonarScanner for MSBuild is distributed as a standalone command line executable, as a extension for <!-- sonarcloud -->[Azure DevOps](/analysis/scan/sonarscanner-for-azure-devops/)<!-- /sonarcloud --><!-- sonarqube -->[Azure DevOps Server](/analysis/scan/sonarscanner-for-azure-devops/)<!-- /sonarqube -->, and as a plugin for [Jenkins](/analysis/scan/sonarscanner-for-jenkins/).
+SonarScanner for .NET is distributed as a standalone command line executable, as a extension for <!-- sonarcloud -->[Azure DevOps](/analysis/scan/sonarscanner-for-azure-devops/)<!-- /sonarcloud --><!-- sonarqube -->[Azure DevOps Server](/analysis/scan/sonarscanner-for-azure-devops/)<!-- /sonarqube -->, and as a plugin for [Jenkins](/analysis/scan/sonarscanner-for-jenkins/).
 
 It supports .Net Core multi-platform projects and it can be used on non-Windows platforms.
```

Comment:
"on every platform (Windows, macOS, Linux)" sounds more fluid, don't you think so? it avoids a negation

Reply:
Makes sense yes ! Thanks.